docker/hq-compose.yml

```diff
@@ -115,6 +115,10 @@ services:
       - "2181:2181"
     volumes:
       - ${VOLUME_PREFIX}${BLANK_IF_TESTS-zookeeper:}/opt/zookeeper-3.4.13/data
+    healthcheck:
+        test: "nc -z zookeeper 2182"
+        interval: 5s
+        timeout: 1m
 
   kafka:
```

Comment:
Does this need a `depends_on` directive since this (not hq-compose-services.yml) is the compose file used by travis tests?

https://github.com/dimagi/commcare-hq/blob/02ac6a443c70104c0342a09d31d62dacf6e437a3/scripts/docker#L144